prowler/providers/aws/aws_provider.py

```diff
@@ -84,8 +90,9 @@ def __init__(
         scan_unused_services: bool = None,
         resource_tags: list[str] = [],
         resource_arn: list[str] = [],
-        audit_config: dict = {},
+        config_file: str = None,
         fixer_config: dict = {},
+        mutelist_path: str = None,
```

Comment:
Thinking about this, maybe we can have `mutelist_path` and `mutelist_content` to allow the caller pass the config directly as an object.

Reply:
I don't get the point of this, inside `__init__` we implement the needed logic to parse the file to a dict so passing directly the dict won't add a value

Reply:
Done